src/index.js

```diff
@@ -517,7 +531,7 @@ function Carotte(config) {
     carotte.handleRetry =
     function handleRetry(qualifier, options, meta = {}, headers, context, message) {
         return err => {
-            return carotte.getChannel(qualifier, options.prefetch)
+            return carotte.getChannel()
```

Comment:
I cleaned up this one because it looked a bit weird. Not sure about the initial implementation.